app/views/evaluation_forms/index.html.erb

```diff
@@ -1,7 +1,27 @@
-<div class="usa-card__container col-md-6">
-  <div class="usa-card__body">
-    <h1>Evaluation Forms</h1>
+<div class="usa-card__body">
+  <h1>Evaluation Forms</h1>
+  <div class="display-flex flex-wrap tablet:flex-justify width-full">
+    <span class="text-base margin-bottom-1">Manage existing evaluation forms and create new evaluation forms.</span>
+
+        <%= link_to new_evaluation_form_path, class: "width-full tablet:width-auto" do %>
+          <button class="usa-button font-body-2xs text-no-wrap" style="background-color: #4d8055">
```

Comment:
let's avoid hard coding styles, especially for colors, and use a class such as `bg-green-primary` or something like that?